.github/workflows/newclient.yaml

```diff
@@ -0,0 +1,60 @@
+name: New Client
+on:
+  schedule:
+    - cron: '0 2 * * *' # nightly at 2 am UTC
+  push:
+    branches: [ main ]
+    paths:
+      - generation/**
+  pull_request:
+    branches: [ main ]
+    paths:
+      - generation/**
+
+  workflow_dispatch:
+
+jobs:
+  generate:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.9'
+          cache: 'pip' # caching pip dependencies
+      - name: Get current date
+        id: date
+        run: echo "::set-output name=date::$(date +'%Y-%m-%d' --utc)"
+      - uses: actions/setup-java@v3
+        with:
+          distribution: zulu
+          java-version: 11
+      - run: java -version
+      - uses: actions/cache@v3
+        id: mvn-cache
+        with:
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-maven-unified-${{ steps.date.outputs.date }}
+      - name: Install dependencies
+        run: pip install -r generation/new_client/requirements.txt
+      - name: Generate
+        run: |
+          python generation/new_client/new-client.py generate \
+              --api_shortname=apikeys \
+              --proto-path=google/api/apikeys \
+              --name-pretty="API Keys API" \
+              --product-docs="https://cloud.google.com/api-keys/" \
+              --api-description="API Keys lets you create and manage your API keys for your projects." \
+              --googleapis-gen-url=https://yoshi-approver:${{ secrets.YOSHI_APPROVER_PRIVATE_TOKEN }}@github.com/googleapis/googleapis-gen.git
+      - name: Push to branch
+        run: |
+          [ -z "`git config user.email`" ] && git config --global user.email "${USERNAME:-script}@google.com"
+          [ -z "`git config user.name`" ] && git config --global user.name "${USERNAME:-script}"
+          git checkout -b newclient_output_${{ github.event_name }}
+          git add --all
+          git commit -m 'feat: re-generated apikeys'
+          git remote add monorepo https://${{ github.actor }}:${{ github.token }}@github.com/${{ github.repository }}.git
+          git fetch -q --unshallow monorepo
+          git push -f monorepo newclient_output_${{ github.event_name }}
+      - name: Test
+        run: mvn test -T C1 -B
```

Comment:
Can you add source code comment that "java-apikeys is included at the aggregator pom.xml"?

Reply:
How about "-f java-apikeys/pom.xml" to skip irrelevant modules?